packages/components/src/card/card/component.js

```diff
@@ -56,7 +56,7 @@ function Card( props, forwardedRef ) {
 	return (
 		<ContextSystemProvider value={ contextProviderValue }>
 			<View { ...otherProps } ref={ forwardedRef }>
-				<View className={ styles.Content }>{ children }</View>
+				<View className={ cx( styles.Content ) }>{ children }</View>
```

Comment:
Here's an example of where we have to awkwardly pass a single serialized style to `cx` to get back a classname. It's not _that_ bad but a limitation to be aware of.

If these are unnacceptable we can selectively convert components to `styled`. That's probably more appropriate for a situation like this anyway.